pkg/cmd/cli/cmd/newapp.go

```diff
@@ -112,9 +109,9 @@ To list all local templates and image streams, use:
 
 To search templates, image streams, and Docker images that match the arguments provided, use:
 
-  %[1]s new-app -S php
```

Comment:
The line above talking about searching "templates, image streams, and Docker images" but the example was different and I decided to update example. Do you think I understood this wrong?